state/selectors/cqfmTestMRLookup.ts

```diff
@@ -0,0 +1,29 @@
+import { selector } from 'recoil';
+import { createCQFMTestCaseMeasureReport } from '../../util/fhir/resourceCreation';
+import { measureBundleState } from '../atoms/measureBundle';
+import { measurementPeriodState } from '../atoms/measurementPeriod';
+import { patientTestCaseState } from '../atoms/patientTestCase';
+
+interface CqfmTestMRLookupType {
+  [id: string]: fhir4.MeasureReport;
+}
+export const cqfmTestMRLookupState = selector<CqfmTestMRLookupType>({
```

Comment:
```suggestion
export const cqfmTestMRLookupState = selector<Record<string, fhir4.MeasureReport>>({
```

Nitpick, but this prob makes more sense as a `Record` type

Reply:
This requires changing the type in the reduce too actually, and deleting the interface